index.html

```diff
@@ -0,0 +1,168 @@
+                80 countries is taking placein october  Lighthouse Meeting Hall
+                South Africa.</p>
+            <p class="date">2023.03.19(SUN) ~ 24(FRI)</p>
+            <p class="last-main-p">@Lighthouse, Global empowerment and more</p>
+          </div> 
+        </section>
+           <!-- main section for the event-->
+    <main class="main-container" id="program">
+        <div class="main-head">
+                <div class="main-box">
+                <h1>Main Program</h1>
+                <div class="line"></div>
+                <div class="event-box">
+                <div class="event">
+                    <img src="images/program1.png" alt="program-img1">
+                    <a href="#">Lecturing</a>
+                    <p>Our youth empowerment programmes tackle youth 
+                        unemployment through skills development. We partner
+                         with organisations who run projects that give young people .</p>
+                </div>
+                <div class="event">
+                    <img src="images/program2.png" alt="program-img2">
+                    <a href="#">Exhibition</a>
+                    <p>Our youth empowerment programmes tackle youth 
+                        unemployment through skills development. We partner
+                         with organisations who run projects that give young people .</p>
+                </div>
+                <div class="event">
+                    <img src="images/program3.png" alt="program-img3">
+                    <a href="#">Forum</a>
+                    <p>Our youth empowerment programmes tackle youth 
+                        unemployment through skills development. We partner
+                         with organisations who run projects that give young people .</p>
+                </div>
+                <div class="event">
+                    <img src="images/program4.png" alt="program-img4">
+                    <a href="#">Workshop</a>
+                    <p>Our youth empowerment programmes tackle youth 
+                        unemployment through skills development. We partner
+                         with organisations who run projects that give young people .
+                    </p>
+                </div>
+                <div class="event">
+                    <img src="images/program5.png" alt="program-img5">
+                    <a href="#">Discussion</a>
+                    <p>Our youth empowerment programmes tackle youth 
+                        unemployment through skills development. We partner
+                         with organisations who run projects that give young people .</p>
+                </div>
+            </div>
+                <a href="#" class="come">See the whole program</a>
+                <button type="submit" class="join">Join Summit 2022</button>
+            </div>
+        </div>
+    </main>
+         <!-- about the speaker session making it dynamic using js-->
```

Comment:
hi there, in the mobile version of this project the following needs to be updated

- The background colour of your design needs to match or have a close similarity to the provided template 
- The alignment of text on the card in the middle need to be updated as well to match the provided template.

the provided designs
![image](https://user-images.githubusercontent.com/60612329/217678967-743be99e-346a-498d-b2e8-89f89fceb67a.png)


![image](https://user-images.githubusercontent.com/60612329/217678894-8038dca5-2c87-4fca-b242-0f78e2a92b00.png)

your design
![image](https://user-images.githubusercontent.com/60612329/217679061-023482ce-be3b-42e7-a55d-cbaebbb422d4.png)

![image](https://user-images.githubusercontent.com/60612329/217679139-d27d2dee-eca2-4b53-afa6-250fba4e3040.png)